File: ESC.Resturant.Data/Entities/CompanyInfo.cs
using ESC.Resturant.Data.Entities;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ESC.Resturant.Data.Entities
{
    public class CompanyInfo
    {
        [Key]
        public int Id { get; set; }
        public string NameAR { get; set; }
        public string NameEN { get; set; }
        public string Phone { get; set; }
        public string Address { get; set; }
        public string Note { get; set; }
        public string Logo { get; set; }
        public virtual ICollection<Branch> Branch { get; set; }

    }
}

File: ESAResturant.NEW.web/ClientApp/src/app/users/Models/CompanyInfo.ts
import { Branch } from "../../admin/setting/Models/Branch";

export class CompanyInfo {

  constructor() {

    this.id = 0;
    this.nameAR = '';
    this.nameEN = '';
    this.phone = '';
    this.address = '';
    this.note = '';
    this.logo = '';
    this.Branches = [];

  }

  id: number;
  nameAR: string;
  nameEN: string;
  phone: string;
  address: string;
  note: string;
  logo: string;
  Branches: Branch[];
}
